tests/python/pants_test/pantsd/test_watchman.py

```diff
@@ -57,12 +57,13 @@ def test_resolve_watchman_path_provided_exception(self):
                                metadata_base_dir=self.subprocess_dir)
 
   def test_maybe_init_metadata(self):
+    # TODO(#7106): is this the right path to patch?
     with mock.patch('pants.pantsd.watchman.safe_mkdir', **self.PATCH_OPTS) as mock_mkdir, \
-         mock.patch('pants.pantsd.watchman.safe_file_dump', **self.PATCH_OPTS) as mock_file_dump:
+         mock.patch('pants.pantsd.watchman.safe_file_dump', **self.PATCH_OPTS) as mock_file_write:
```

Comment:
Should this go back to being `mock_file_dump`?

Reply:
Thanks for the catch!!! Done!